django/contrib/gis/geoip2.py

```diff
@@ -173,9 +174,10 @@ def __repr__(self):
     def _check_query(self, query, city=False, city_or_country=False):
         "Check the query and database availability."
         # Making sure a string was passed in for the query.
-        if not isinstance(query, str):
+        if not isinstance(query, (str, ipaddress.IPv4Address, ipaddress.IPv6Address)):
```

Comment:
Update the comment above.

Reply:
I think we can drop the comment. It doesn't add value - the `isinstance(...)` is quite clear enough.